sdk/go/client.go

```diff
@@ -1,16 +1,16 @@
-//go:generate go run ../../../cmd/dagger client-gen -o ./api/api.gen.go
```

Comment:
Guessing this had to change because `go run` doesn't work across module boundaries?

Is this the new flow for regenerating the SDK?


```sh
go install ./cmd/dagger
go generate ./...
```

If so that's fine (I assumed it already worked that way). Or does `go generate` similarly not traverse across module boundaries?

Side note: since we have `replace github.com/dagger/dagger => ../` in the SDK `go.mod`, I wonder if this works?:

```sh
//go:generate go run github.com/dagger/dagger/cmd/dagger`? :thinking: 
```

If the steps are anything more complicated than above I humbly request a script. :)

Reply:
> Guessing this had to change because go run doesn't work across module boundaries?

Correct

> Is this the new flow for regenerating the SDK?

Worse.

```
cd sdk/go
go generate ./...
```

`./..` doesn't cross the go.mod boundary

> Side note: since we have replace github.com/dagger/dagger => ../ in the SDK go.mod, I wonder if this works?:

Ah! Perhaps! Let me try

Reply:
> If the steps are anything more complicated than above I humbly request a script. :)

Yes. Right now it looks silly, but `go generate` wasn't going to work with the Python and Node SDK anyway, so we needed a script anyway.

There's still time to back down from this sub-module if it becomes too annoying :)